docs/cli/cmdline/script/run.md

```diff
@@ -0,0 +1,51 @@
+---
+title: terramate script run - Command
+description: With the terramate
+
+prev:
+  text: 'Script List'
+  link: '/cli/cmdline/script/list'
+
+next:
+  text: 'Script Tree'
+  link: '/cli/cmdline/script/tree'
+---
+
+# Script Run
+
+**Note:** This is an experimental command that is likely subject to change in the future.
+
+The `script run SCRIPT-NAME` command will run a Terramate script over a set of stacks. The script will run on all stacks under the current working directory where:
```

Comment:
```suggestion
The `script run SCRIPT-NAME...` command will run a Terramate script over a set of stacks. The `SCRIPT-NAME` needs to match the labels defined in the `script` block.

The script will run on all stacks under the current working directory where:
```

Reply:
what's the meaning of the "..."? normally that indicates multiple words, but as far as I can see `script run` only supports one script. If the intention was that SCRIPT-NAME can have spaces, I think "..." will only be confusing.

If the intention is that we will support labels other than name, then perhaps it should be changed to `script run SCRIPT-LABEL...`

Reply:
> If the intention is that we will support labels other than name, then perhaps it should be changed to script run SCRIPT-LABEL...

we do support labels and this is how it's supposed to be used.

The script definition below:
```
script "project" "deploy" {
...
}
```

can only be executed by:
```
$ terramate run script project deploy
```

The labels in the script definition declare a command, similar to `terramate experimental metadata` and `terramate experimental globals`.

So in my opinion, some hint must be put in the docs to make this clear.

Reply:
Another example:

```
script "multi word name" "another multi word name" {
...
}
```

this script can only be executed by:
```
$ terramate script run "multi word name" "another multi word name"
```
if you remove the quotes, it will NOT work... because we refer the `script` labels in the command-line to lookup the script.

Reply:
I've changed it to what we agreed on in Slack, thanks